src/components/Carousel.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import Slider from 'react-slick';
+import { useNavigate } from 'react-router-dom';
+import './Carousel.css';
+
+type Item = {
+  id: number;
+  title: string;
+  poster_path: string;
+};
+
+type Person = {
+  id: number;
+  name: string;
+  profile_path: string;
+};
+
+type Props = {
+  text: string;
+  type: string;
+};
+
+const Carousel: React.FC<Props> = ({ text, type }) => {
+  const API_TOKEN = process.env.REACT_APP_TMDB_TOKEN;
+  const [popular, setPopular] = useState<Item[]>([]);
+  const [people, setPeople] = useState<Person[]>([]);
+  const settings = {
```

Comment:
Please move setting outside of a component. It will make it easier to read and the object will be created only once and not recreated on every re-render.
If it is used only in this component, it can stay at the top. If it would be reused or if you have many config settings, then better to declare it in a separate file and import.